services/worker-manager/schemas/v1/worker-full.yml

```diff
@@ -80,7 +80,7 @@ properties:
     type: integer
     minimum: 1
 
-additionalProperties: false
+additionalProperties: true
```

Comment:
Do you remember how it was used before and why is it needed to allow extra props now?

Reply:
This was needed in the two new wm APIs, `listWorkers` and `getWorker` because they both return a _little_ more data (from the `queue_workers` table) because of that join happening. I decided to just use the same schema that was there and enabled additional props. I could separate this out into a new schema for just those two new APIs that have the queue data.